app/controllers/making_a_differences_controller.rb

```diff
@@ -0,0 +1,59 @@
+class MakingADifferencesController < ApplicationController
+  before_action :set_making_a_difference, only: [:show, :edit, :update, :destroy]
+
```

Comment:
You need `load_and_authorize_resource` here so that people who aren't logged in can't try to manage the system.

Reply:
I can really spell :( added the CanCan load_and_authorize_resource, not expect but except